pkg/controller/common/certificates/dynamic_watches.go

```diff
@@ -18,18 +18,27 @@ func CertificateWatchKey(namer name.Namer, ownerName string) string {
 
 // reconcileDynamicWatches reconciles the dynamic watches needed by the HTTP certificates.
```

Comment:
that's not only the http cert now, right?

Reply:
No it still is. I use the new function `ReconcileCustomCertWatch` for transport (which this function also calls). The latter needs a godoc though, so 👍